### DISCUSSION TITLE
Oscillation

### DISCUSSION THREAD

**Discussion by Patrick3313** [flybywiresim/aircraft#2467] [Help]
Dear all,

I install thelast stable version of the A320nx on FS 2020 , unfortunatly I always have bank oscillation , also the aircraft does not follow the green flight plan on MFD .
Does somebody have an idea of what is going wrong?

Thank you for your help

Patrick

**Comment by maxpull82** [comment]
confermo! anche sull' ultima versione stesso problema. decollo attivazione pilota automatico automatico e l'aereo impazzisce.
E poi segnalo anche la revisione dell' mcdu tante opzioni non sono attive e problemi sull'inserimento dei punti rotta 

Massimiliano

**Reply by Patrick3313** [reply]
Hello Massimiliano,

Thanl you foe reply, I feel not alone now !

Regards

Patrick

**Comment by hotshotp** [comment]
unfortunately that is an MSFS issue that happens for all airliners

**Reply by maxpull82** [reply]
speriamo risolvano presto anche questo problema. perchè e' noioso come problema intanto suggesisco di mettere "sec 1" su off comunque ho notato che il problema si presenta con le livree.

**Reply by Patrick3313** [reply]
Hello Massimiliano,

Where do you deactivate "sec 1"? please,

**Reply by hotshotp** [reply]
the middle left of the overhead panel in the flight control section

**Reply by maxpull82** [reply]
ciao certamente ti allego la foto :) 
![Overhead_panel_of_an_Airbus_A320_during_cruise](https://user-images.githubusercontent.com/75287235/102501115-5c62ca80-407d-11eb-90e1-a2e6e75e9eed.jpg)

**Reply by Patrick3313** [reply]
Thank you!
